python_modules/dagster/dagster/_cli/api.py

```diff
@@ -573,6 +584,28 @@ def _execute_step_command_body(
     "code from this server.",
     envvar="DAGSTER_CONTAINER_CONTEXT",
 )
+@click.option(
+    "--load-env-vars",
+    is_flag=True,
+    required=False,
+    default=False,
+    help="Whether to load env vars from the instance and inject them into the environment.",
+    envvar="DAGSTER_LOAD_ENV_VARS",
+)
```

Comment:
nit: should the name here align with the instance component more directly? `inject-secrets` ? Not public so less critical - maybe at least refer to `SecretsLoader` in the help text